src/languages.hpp

```diff
@@ -233,7 +235,8 @@ inline const std::map<Key, const char *> germanLan{
     {Key::German, "Deutsch"},
     {Key::Ready, "Bereit"},
     {Key::FontSize, "Schriftgröße"},
-    {Key::SettingsText, "Legen sie hier ihre persönlichen Einstellungen fest"}};
+    {Key::SettingsText, "Legen sie hier ihre persönlichen Einstellungen fest"},
+    {Key::saving, "sparen ..."}};
```

Comment:
```suggestion
    {Key::saving, "speichern ..."}};
```

"sparen" means something like saving money in english.